examples/gnrc_border_router/Makefile

```
@@ -7,6 +7,18 @@ BOARD ?= samr21-xpro
 # This has to be the absolute path to the RIOT base directory:
 RIOTBASE ?= $(CURDIR)/../..
 
+# Default to using ethos for providing the uplink when not on native
+UPLINK ?= ethos
+
+# Check if the selected Uplink is valid
+ifeq (,$(filter ethos slip wifi,$(UPLINK)))
+  $(error Supported Uplinks are ethos, slip and wifi)
```

```suggestion
  $(error Supported uplinks are `ethos`, `slip` and `wifi`)
```

a) gramar 
b) mark the possible values as such (you can also use `'` if you rather prefer that).